packages/gatsby/cache-dir/ssr-develop-static-entry.js

```diff
@@ -9,7 +9,7 @@ import { grabMatchParams } from "./find-path"
 import syncRequires from "$virtual/ssr-sync-requires"
 
 import { RouteAnnouncerProps } from "./route-announcer-props"
-import { ServerLocation, Router, isRedirect } from "@reach/router"
+import { ServerLocation, Router, isRedirect } from "@gatsbyjs/reach-router"
```

Comment:
This uses our alias `@reach/router` > `@gatsbyjs/reach-router`, clarify import now since we'll remove the alias in Gatsby 5